intranet/settings/__init__.py

```diff
@@ -784,7 +787,12 @@ def get_log(name):  # pylint: disable=redefined-outer-name; 'name' is used as th
         "schedule": FCPS_EMERGENCY_CACHE_UPDATE_INTERVAL,
         "args": (),
     },
-    "reset-routes": {"task": "intranet.apps.bus.tasks.reset_routes", "schedule": celery.schedules.crontab(hour=8, minute=0), "args": ()},
+    "reset-routes-afternoon": {"task": "intranet.apps.bus.tasks.reset_routes", "schedule": celery.schedules.crontab(hour=8, minute=0), "args": ()},
```

Comment:
```suggestion
    "reset-routes-afternoon": {"task": "intranet.apps.bus.tasks.reset_routes", "schedule": celery.schedules.crontab(hour=0, minute=0), "args": ()},
```

I'd rather reset the routes earlier. Security might pull up the page at 7:50 and be confused that the routes are still there.

And since other parts of this PR assume that it rolls over at midnight, that's a convenient time.